pages/sdk/protocol-kit/guides/safe-deployment/deploy-safe.mdx

```diff
@@ -0,0 +1,139 @@
+import { Steps } from 'nextra/components'
+
+# Deploy a Safe
+
+In this guide, you will learn how to deploy a new Safe using the Protocol Kit. This process includes initializing the Protocol Kit, setting up your Safe configuration, and executing the deployment.
+
+For more detailed information, see the [Protocol Kit Reference](../../../protocol-kit/reference/safe.mdx).
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install dependencies
+
+First, you need to install the Protocol Kit.
+
+```bash
+pnpm add @safe-global/protocol-kit
+```
+
+## Steps
+
+<Steps>
+
+  ### Imports
+
+  Begin by importing the `Safe` class from the Protocol Kit.
+
+  {/* <!-- vale off --> */}
+
+  ```typescript
+  import Safe from '@safe-global/protocol-kit'
+  ```
+
+  {/* <!-- vale on --> */}
+
+  ### Create a signer
+
+  Firstly, you need to get a signer, this example uses a private key, but any way to get an `EIP-1193` compatible signer can be used.
+
+  {/* <!-- vale off --> */}
+
+  ```typescript
+  const SIGNER_PRIVATE_KEY = // ...
+  const RPC_URL = 'https://rpc.ankr.com/eth_sepolia'
+  ```
+
+  {/* <!-- vale on --> */}
+
+  ### Initialize the `ProtocolKit`
+
+  {/* <!-- vale off --> */}
+
+  Initialize the Protocol Kit with the `provider`, `signer`, and the `predictedSafe` configuration.
```

Comment:
I will ellaborate a little more here as well. Why we use the predictedSafe prop for example and link to the configuration in the reference guide

Reply:
done!